README.md

```diff
@@ -2,19 +2,29 @@
 
 ## Description
 A mental health app for developers.
+The app allows users to search for videos about a mental health topic to help manage work-related stress.
 
 ## Installation
+Foundation
+Youtube Data API
+Youtube Player API
+Local Storage
+jQuery
 
-N/A
+
+![Screenshot 2023-02-01 190042](https://user-images.githubusercontent.com/119261811/216206069-e16aff3b-b4cf-4e14-93e7-ee7d1cc867eb.png)
 
 ## Usage
 
-![Website Screenshot]
+file:///C:/Users/crist/bootcamp/DevelopMental/index.html
 
-[Deployed Website Link]
 
 ## Credits
```

Comment:
Here's a suggestion about what else to include in the credits section: 
If you used any third-party assets that require attribution, list the creators with links to their primary web presence in this section.
If you followed tutorials, include links to those here as well.